tests/objc-cli/libmanaged/Tests/Tests.m

```diff
@@ -413,6 +413,18 @@ - (void)testTypes {
     XCTAssertEqualObjects ((__bridge NSString *) nil, [Type_String nullString], "null string");
     XCTAssertEqualObjects (@"", [Type_String emptyString], "empty string");
     XCTAssertEqualObjects (@"Hello World", [Type_String nonEmptyString], "non-empty string");
+
+	NSDecimalNumber * decimalzero = [Type_Decimal zero];
+	NSDecimalNumber * nsdecimalzero = [NSDecimalNumber zero];
+	XCTAssertEqualObjects (nsdecimalzero, decimalzero, "NSDecimalNumber zero");
+
+	NSDecimalNumber * decimalmin = [Type_Decimal min];
```

Comment:
Maybe a few non-boundary cases as well? -1, 1, 42 along with some with with fractional parts.

Also, you have tabs while above has spaces. Maybe fix rest of test method?